docs/sources/tempo/operations/monitor/set-up-monitoring.md

```diff
@@ -63,265 +63,5 @@ In your Grafana instance, you'll need:
 * [A Tempo data source](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/configure-tempo-data-source/) (created in the previous section)
 * A [Mimir (Prometheus) data source](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/datasources/prometheus/)
 
-## Enable Tempo metrics scraping
 
-Tempo exposes Prometheus metrics from all of its components to allow meta-monitoring.
-Alloy can both collect traces and act as a Prometheus scraper to retrieve metrics from Tempo.
-To use this capability, you need to configure Alloy to scrape from all of the components.
-
-Alloy lets you discover targets to scrape in a cluster via a variety of ways.
-Usually for Prometheus metrics scraping, you would annotate the pods, services, and others, to signal that Alloy should scrape metrics from those objects using annotations such as `prometheus/scrape: true` as well as a port and path.
-
-However, the Tempo objects already have some convenient annotations supplied under the `app.kubernetes.io` prefixes.
-The Helm deployment includes these annotations.
-For example, the label annotations for Tempo’s distributor component contain:
-
-```yaml
-app.kubernetes.io/component=distributor
-app.kubernetes.io/instance=tempo
-app.kubernetes.io/managed-by=Helm
-app.kubernetes.io/name=tempo
-app.kubernetes.io/part-of=memberlist
-app.kubernetes.io/version=2.6.0
-```
-
```

Comment:
Is all of this information already in the K8s monitoring documentation?

Reply:
The label selectors section is the "rough" comparison to this.

Reply:
Perfect. Thank you.